docs/cli/graphql-transformer/directives.md

```diff
@@ -19,19 +19,35 @@ top level object types in your API that are backed by Amazon DynamoDB.
 
 ```
 directive @model(
-  queries: ModelQueryMap,
-  mutations: ModelMutationMap,
+  queries: ModelQueryMap
+  mutations: ModelMutationMap
   subscriptions: ModelSubscriptionMap
+  timestamps: TimestampMap
 ) on OBJECT
-input ModelMutationMap { create: String, update: String, delete: String }
-input ModelQueryMap { get: String, list: String }
+input ModelMutationMap {
+  create: String
+  update: String
+  delete: String
+}
+input ModelQueryMap {
+  get: String
+  list: String
+}
 input ModelSubscriptionMap {
   onCreate: [String]
   onUpdate: [String]
   onDelete: [String]
   level: ModelSubscriptionLevel
 }
-enum ModelSubscriptionLevel { off public on }
+enum ModelSubscriptionLevel {
+  off
+  public
+  on
+}
+input TimestampMap {
+  create: String
```

Comment:
I think the question for API review is should this be 
```
input TimestampMap {
  createdOn: String
  updatedOn: String
}
```

Just to make it more explicit what you're renaming.